src/py/aspen/database/models/entity.py

```diff
@@ -123,3 +123,29 @@ def get_children(
                 results.append(tup)
 
         return results
+
+    def accessions(self) -> Sequence[Accession]:
+        from .accessions import Accession
+
+        results: MutableSequence[Accession] = list()
+        for workflow, accessions in self.get_children(Accession):
+            results.extend(accessions)
+        return results
+
+    def add_accession(
+        self,
+        repository_type: PublicRepositoryType,
+        public_identifier: str,
+        workflow_start_datetime: Optional[datetime.datetime] = None,
+        workflow_end_datetime: Optional[datetime.datetime] = None,
+    ):
+        """Adds an accession to this object."""
+        from .accessions import Accession
+
+        Accession.add_to_entity(
+            self,
+            repository_type,
+            public_identifier,
+            workflow_start_datetime,
+            workflow_end_datetime,
+        )
```

Comment:
i also find this a little confusing (passing self as the entity for add_to_entity), and the whole entity/ accession relationship, because i read this like add an accession for this entity and then add the accession to this entity. Maybe it's the end of the day and i'm tired though lol

Reply:
It's add an accession for this entity.  How about "create_for_entity"?  It's not exactly accurate either because it's not just creating it, it's attaching it to the entity.

Reply:
what about `attach_to_entity`? for some reason that seems more clear to me

Reply:
Updated.